packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/actionTypes.js

```diff
@@ -18,3 +18,7 @@ export const SET_VERIFY_ERROR = '@COMPONENT.SET_VERIFY_ERROR'
 export const SUBMIT_MICRO_DEPOSITS = '@COMPONENT.SUBMIT_MICRO_DEPOSITS'
 
 export const SUBMIT_QUOTE = '@COMPONENT.SUBMIT_QUOTE'
+export const SUBMIT_SELL_QUOTE = '@COMPONENT.SUBMIT_SELL_QUOTE'
+
+export const SET_TRADE_ERROR = '@COMPONENT.SET_TRADE_ERROR'
+export const CLEAR_TRADE_ERROR = '@COMPONENT.CLEAR_TRADE_ERROR'
```

Comment:
Can we try to use the Remote pattern instead like here: https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/329/commits/f81eb4f438fac6638e531c84d21516c76c09ccc1#diff-b1b5fd43828e8dc31156198f97acdff3R23. That way you could get rid of SFOX_SIGNUP_FAILURE, CLEAR_SIGNUP_ERROR, SET_VERIFY_ERROR, SET_TRADE_ERROR, and CLEAR_TRADE_ERROR